alire_common.gpr

```diff
@@ -31,7 +31,14 @@ abstract project Alire_Common is
             "-gnatyu",   -- no unnecessary blank lines
             "-gnatyx",   -- no extra parens around conditionals
             "-gnaty-s"); -- relax fwd decl
-      when "disabled" => Style_Check_Switches := ();
+      when "disabled" => Style_Check_Switches :=
+           ("-gnat2022",
```

Comment:
I don't think it's good to change language version when disabling style checks.
I fear this is going to have unexpected consequences that will make your life harder.